google-cloud-clients/google-cloud-logging/README.md

```diff
@@ -147,7 +147,7 @@ Then, to list the log entries, use the following code:
 ``` java
 Page<LogEntry> entries = logging.listLogEntries(
     EntryListOption.filter("logName=projects/" + options.getProjectId() + "/logs/test-log"));
-Iterator<LogEntry> entryIterator = entries.iterateAll();
+Iterator<LogEntry> entryIterator = entries.iterateAll().iterator();
```

Comment:
how come `entries.iterateAll()` & `entries.iterateAll().iterator()`; gives same output.

Reply:
Ignore this.

Reply:
@pmakani I thought this is java file and was trying to understand why `entries.iterateAll()` & `entries.iterateAll().iterator()` gives same output.